src/clientManager.js

```diff
@@ -160,5 +161,5 @@ ClientManager.prototype.autoload = function(/* sockets */) {
 				log.info("User '" + name + "' disconnected");
 			}
 		});
-	}, 1000);
+	}, 1000, {maxWait: 10000}));
```

Comment:
What does maxWait do? Does it stop watching after that time, or does it reload after that time whether it has changed or not?